src/plot_utils.py

```diff
@@ -633,7 +657,7 @@ def zmatrix(data_df, clustermap_kws=None, row_ordering=None,
         df = df.ix[:, col_ordering]
         df = df.ix[row_ordering, :]
         del clustermap_kws['pivot_kws']
-        fig, ax = plt.subplots()
+        _fig, ax = plt.subplots()
```

Comment:
Presumably the rename is to indicate it is explicitly unused.

But it looks like there's a separate issue here: this function appears to be using global state, which is wrong.  What state is this reading, and what is writing that state?  It's not obvious to me from cursory examination of the surrounding code.